REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,292 @@
+
+To demonstrate how we configure this policy, we'll use the `WifiPeriodicReplicationReceiver` to trigger our replications
+when the device is connected to Wifi.
+
+###BroadcastReceiver
+Our subclass of `WifiPeriodicReplicationReceiver` is very simple and only needs to configure the name of the subclass of `PeriodicReplicationService` that we want our `BroadcastReceiver` to interact with by creating a default constructor that
+passes it to the superclass's constructor. We'll be calling our service `MyReplicationService`, so our `BroadcastReceiver` looks like:
+
+```java 
+public class MyWifiPeriodicReplicationReceiver extends WifiPeriodicReplicationReceiver<MyReplicationService> {
+
+    public MyWifiPeriodicReplicationReceiver() {
+        super(MyReplicationService.class);
+    }
+
+}
+```
+
+###Service
+Our service must configure the name of our `BroadcastReceiver` (i.e. `MyWifiPeriodicReplicationReceiver`) and does this
+in the default constructor by passing the class's name to the superclass constructor. This allows the super class to invoke
+`MyWifiPeriodicReplicationReceiver` when the periodic replications are due. We must also implement the abstract methods:
+
+```java
+public class MyReplicationService extends PeriodicReplicationService {
+
+    public static int PUSH_REPLICATION_ID = 0;
+    public static int PULL_REPLICATION_ID = 1;
+
+    private static final String TAG = "MyReplicationService";
+
+    private static final String TASKS_DATASTORE_NAME = "tasks";
+    private static final String DATASTORE_MANGER_DIR = "data";
+
+    public MyReplicationService() {
+        super(MyWifiPeriodicReplicationReceiver.class);
+    }
+
+    @Override
+    protected Replicator[] getReplicators(Context context) {
+        try {
+            URI uri = new URI("https", "my_api_key:my_api_secret", "myaccount.cloudant.com", 443, "/" + "mydb", null, null);;
+
+            File path = context.getApplicationContext().getDir(
+                DATASTORE_MANGER_DIR,
+                Context.MODE_PRIVATE
+            );
+
+            DatastoreManager manager = new DatastoreManager(path.getAbsolutePath());
+            Datastore datastore = null;
+            try {
+                datastore = manager.openDatastore(TASKS_DATASTORE_NAME);
+            } catch (DatastoreNotCreatedException dnce) {
+                Log.e(TAG, "Unable to open Datastore", dnce);
+            }
+
+            Replicator pullReplicator = ReplicatorBuilder.pull().from(uri).to(datastore).withId(PULL_REPLICATION_ID).build();
+            Replicator pushReplicator = ReplicatorBuilder.push().to(uri).from(datastore).withId(PUSH_REPLICATION_ID).build();
+
+            return new Replicator[]{pullReplicator, pushReplicator};
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Override
+    protected int getBoundIntervalInSeconds() {
+        return 5 * 60; // 5 minutes
```

Comment:
Android does have `java.util.concurrent.TimeUnit`, so it could be done as you suggest (which I agree is quite nice), but in the context of this example, I'm not sure it will make it more readable, particularly for people unfamiliar with `TimeUnit`. The current implementation doesn't preclude people from doing it your way in real code if they want to (although they'd have to cast the return value to `int`).